LandSandBoat/server
# [chore] Replace int8* "strings" with std::string

**_I affirm:_**

## What does this pull request do?

Replace int8* "strings" with std::string
## Steps to test these changes

Linkshells:
check lsmes, upgrade/downgrade pearlsack, kick, get linkshell list
check equipped name on player via /check

Parties:
/pcmd leader, /pcmd kick, /acmd kick, pcmd leader, get party list

Item signatures:
signing items, /check on signed items displaying, /check in bazaar items displaying

ZMQ messages:
/p, /l messages, `gm CharacterName 5` command in map console

Lua stuff:
get name of pet, get name of ability, no errors talking to NPCs, using abilities, pets using abilities, 

use /tells

Things I haven't covered, but are also used in general gameplay/startup.
